roqoqo-qasm/src/parser.rs

```diff
@@ -162,6 +167,44 @@ fn gate_dispatch(name: &str, params: &[String], qubits: &[usize]) -> Option<Oper
                 CalculatorFloat::ZERO,
             )))
         }
+        "u2" => {
+            let theta = CalculatorFloat::ZERO;
```

Comment:
I think theta has to be `pi/2`, but double check the definitions here https://arxiv.org/pdf/1707.03429